src/modules/UI/components/WalletListModal/WalletListModal.ui.js

```diff
@@ -20,8 +20,8 @@ export default class WalletListModal extends Component<Props> {
     const top = this.props.topDisplacement ? this.props.topDisplacement : 38
     return (
       <Animatable.View style={[b(), styles.topLevel, {position: 'absolute', top: top, height: (platform.deviceHeight - platform.toolbarHeight - platform.footerHeight)}]}
-        animation='fadeInDown'
-        duration={100} >
+        animation='fadeInUp'
```

Comment:
what does this change do?

Reply:
Another animation change, sorry did not feature branched it. Did that before I was briefed